os/net/netmgr/netmgr_ioctl_lwip.c

```diff
@@ -182,6 +182,28 @@ int _netdev_dhcpc_stop(const char *intf)
 
 	return OK;
 }
+
+
+int _netdev_dhcpc_sethostname(struct lwip_dhcp_msg *msg)
+{
+#ifdef CONFIG_LWIP_DHCP_HOSTNAME
+  // it's blocked call. So it's ok to use local variable
```

Comment:
will use /* for comment lines?

Reply:
There is no coding rule about it. So It seems ok.